spacy_llm/backends/rest/backend/base.py

```diff
@@ -24,12 +26,14 @@ def __init__(
             be raised.
             Note that only response object structure will be checked, not the prompt response text per se.
         max_tries (int): Max. number of tries for API request.
-        timeout (int): Timeout for API request in seconds.
+        interval (int): Time interval (in seconds) for API retries in seconds.
```

Comment:
```suggestion
        interval (int): Time interval (in seconds) for API retries in seconds. We implement a base 2 exponential backoff at each retry.
```

The wording is far from perfect, but that seems like relevant information?

By the way, we could also accept a [Thinc schedule](https://thinc.ai/docs/api-schedules#compounding) (most probably overkill, but still)

Reply:
Updated in https://github.com/explosion/spacy-llm/pull/110/commits/f1da1268e94308420b2d90e72540ccdf02fc39cb. Thinc schedules are probably overkill if we just need this one schedule (if that changes, it'd make sense).